uk.co.compucorp.civicrm.tasksassignments/gulpfile.js

```diff
@@ -17,7 +20,20 @@ gulp.task('sass', function (done) {
   });
 });
 
-gulp.task('requirejs-bundle', function (done) {
+gulp.task('js-crm', function () {
```

Comment:
`js-crm` doesn't give me an idea of what's going on here. Why not `crm-js-bundle` ?